Example/Example/ValueViewController.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+
+class ValueViewController: UIViewController {
+
+    @IBOutlet weak var valueLabel: UILabel!
+
+    var changingValue: ((Int) -> Void)!
+    var presentModal: (() -> Void)!
+
+    @IBAction func changeValue() {
+        guard
+            let valueText = valueLabel.text,
+            let value = Int(valueText)
```

Comment:
Forcing integer from text seems scary... How about a `try`?

Reply:
We are not forcing it, there is a guard there if it should fail.

Reply:
ohh thats right, I always forget about that it's more like a matching thing in guards :)